images/macos/toolsets/toolset-13.json

```diff
@@ -75,7 +75,8 @@
             "gmp",
             "r",
             "yq",
-            "unxip"
+            "unxip",
+            "xcodesorg/made/xcodes"
```

Comment:
I kinda do not like Xcode's to be involved, the tool is way too unpredictable, can we get rid of it and simply download the platform separately with xcodebuild?

Reply:
I kinda not like too, but xcodebuild just timed out downloading that particular SDK